src/main/java/by/exadel/internship/service/impl/FormServiceImpl.java

```diff
@@ -32,13 +33,13 @@
 @Slf4j
 public class FormServiceImpl implements FormService {
 
+    private static final String SIMPLE_CLASS_NAME = FormService.class.getSimpleName();
```

Comment:
Why do you need it. Looks strange

Reply:
that's not mine. Nikita did it for log4j